### PR TITLE
Feature/new method is undefined 

### PR DESCRIPTION
**Pull request by MelleD**
New method is empty for JsonNullable
#7

### CODE REVIEW

**Comment by cbornet** [comment]
Thanks for the PR. I think your previous naming of `isUndefined` was more coherent with the rest of the API. I chose `undefined` instead of `empty` to avoid confusion with Optional.
Can you rename to `isUndefined` ?

**Comment by SirWayne** [comment]
@cbornet done

File: src/test/java/org/openapitools/jackson/nullable/ModuleTestBase.java [hunk 1]
```diff
@@ -45,4 +45,17 @@ protected String quote(String str) {
     protected String aposToQuotes(String json) {
         return json.replace("'", "\"");
     }
+
+    private void assertValue(JsonNullable<?> jsonNullable, boolean present){
+        assertEquals(jsonNullable.isPresent(), present);
+        assertEquals(jsonNullable.isUndefined(), !present);
+    }
+
+    protected void assertValueIsPresent(JsonNullable<?> jsonNullable){
```
Comment (cbornet) [inline src/test/java/org/openapitools/jackson/nullable/ModuleTestBase.java:54]: Maybe just `assertPresent` ?

File: src/test/java/org/openapitools/jackson/nullable/ModuleTestBase.java [hunk 2]
```diff
@@ -45,4 +45,17 @@ protected String quote(String str) {
...
+        assertValue(jsonNullable, true);
+    }
+
+    protected void assertValueIsEmpty(JsonNullable<?> jsonNullable){
```
Comment (cbornet) [inline src/test/java/org/openapitools/jackson/nullable/ModuleTestBase.java:58]: Replace with `assertUndefined`

File: src/test/java/org/openapitools/jackson/nullable/PolymorphicJsonNullableTest.java [hunk 3]
```diff
@@ -31,7 +31,7 @@ public void testPolymorphic14() throws Exception
 
         final Container fromJson = MAPPER.readValue(json, Container.class);
         assertNotNull(fromJson.contained);
-        assertTrue(fromJson.contained.isPresent());
+        assertValueIsPresent(fromJson.contained );
```
Comment (cbornet) [inline src/test/java/org/openapitools/jackson/nullable/PolymorphicJsonNullableTest.java:34]: Space after `contained` to be removed

**Review by cbornet — CHANGES_REQUESTED** [review]

**Comment by cbornet** [comment]
@MelleD I totally forgot about this PR ! I'm so very sorry ! 
I did some comments and it needs to be rebased. Do you still want to work on it ?

**Pull request closed without merging**